{
  "Header": {
    "Original": "CVE-2022-31291 (An issue in dlt_config_file_parser.c of dlt-daemon v2.18.8 allows atta ...)",
    "Line": 32419,
    "ID": "CVE-2022-31291",
    "Description": "(An issue in dlt_config_file_parser.c of dlt-daemon v2.18.8 allows atta ...)"
  },
  "Annotations": [
    {
      "Original": "- dlt-daemon 2.18.6-2.1 (bug #1014534)",
      "Line": 32420,
      "Type": "package",
      "Package": "dlt-daemon",
      "Kind": "fixed",
      "Version": "2.18.6-2.1",
      "BugNo": 1014534
    },
    {
      "Original": "[bullseye] - dlt-daemon 2.18.6-1+deb11u1",
      "Line": 32421,
      "Type": "package",
      "Release": "bullseye",
      "Package": "dlt-daemon",
      "Kind": "fixed",
      "Version": "2.18.6-1+deb11u1"
    },
    {
      "Original": "[buster] - dlt-daemon \u003cno-dsa\u003e (Minor issue)",
      "Line": 32422,
      "Type": "package",
      "Release": "buster",
      "Package": "dlt-daemon",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://github.com/COVESA/dlt-daemon/pull/376",
      "Line": 32423,
      "Type": "NOTE",
      "Description": "https://github.com/COVESA/dlt-daemon/pull/376"
    },
    {
      "Original": "NOTE: https://github.com/COVESA/dlt-daemon/commit/6a3bd901d825c7206797e36ea98e10a218f5aad2",
      "Line": 32424,
      "Type": "NOTE",
      "Description": "https://github.com/COVESA/dlt-daemon/commit/6a3bd901d825c7206797e36ea98e10a218f5aad2"
    }
  ]
}